src/Bundle/DependencyInjection/DoctrineBehaviorsExtension.php

```diff
@@ -16,5 +17,21 @@ public function load(array $configs, ContainerBuilder $container)
     {
         $loader = new YamlFileLoader($container, new FileLocator(__DIR__.'/../../../config'));
         $loader->load('orm-services.yml');
+
+        $configuration = new Configuration();
+        $config = $this->processConfiguration($configuration, $configs);
+
+        foreach ($config as $behavior => $enabled) {
+            $container->setParameter(sprintf('knp_doctrine_behaviors.%s.enabled', $behavior), $enabled);
```

Comment:
I don't think we need to create a parameter to store if enabled. The presence of the service is enough.